### PR TITLE
Add login for admins and add SerjB to contributors

### PR DESCRIPTION
**Pull request by bhkhdh**

### CODE REVIEW

File: README.rdoc [hunk 1]
```diff
@@ -1,5 +1,6 @@
 == README
-
```
Comment (rakvium) [inline README.rdoc:2]: Please add Contributors header using Markdown: [https://guides.github.com/features/mastering-markdown/](https://guides.github.com/features/mastering-markdown/)

File: README.rdoc [hunk 2]
```diff
@@ -1,5 +1,6 @@
...
+Serj_Bichkhidzhy 
+serj.bichkhidzhy@gmail.com
```
Comment (rakvium) [inline README.rdoc]: Please move your email to the line with your name :construction_worker_man:

**Review by rakvium — CHANGES_REQUESTED** [review]
Please fix formatting issues (see my comments)

**Comment by SerjB** [comment]
Done

File: README.rdoc [hunk 3]
```diff
@@ -1,5 +1,7 @@
 == README
 
+Serj_Bichkhidzhy 	serj.bichkhidzhy@gmail.com
```
Comment (rakvium) [inline README.rdoc]: @SerjB please add Contributors section above, like `## Contributors`.
Also please don't use tabs, spaces are more consistent.

File: README.rdoc (hunk 2 above)
Comment (bhkhdh) [reply README.rdoc]: Done

File: app/models/application_record.rb [hunk 4]
```diff
@@ -1,3 +1,7 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+<<<<<<< HEAD
 end
+=======
```
Comment (rakvium) [inline app/models/application_record.rb]: Please remove unnecessary lines.

**Review by rakvium — CHANGES_REQUESTED** [review]

File: README.rdoc [hunk 5]
```diff
@@ -1,8 +1,8 @@
 == README
 
-==== Contributors:
+Contributors.
```
Comment (rakvium) [inline README.rdoc]: Why there is a dot here?

**Review by rakvium — APPROVED** [review]

File: app/models/admin.rb [hunk 6]
```diff
@@ -0,0 +1,6 @@
+class Admin < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
```
Comment (rakvium) [inline app/models/admin.rb]: Admin shouldn't be registerable.

**Review by rakvium — CHANGES_REQUESTED** [review]

**Review by rakvium — APPROVED** [review]

**Pull request merged**